### DISCUSSION TITLE
WSA NETWORK ISSUE

### DISCUSSION THREAD

**Discussion by Aayush1914** [community/community#77096] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why WSA  give network error ,  and  unable to connect pc wifi.. @mustardcheff @mustardchef

**Comment by zoelabbb** [comment]
maybe you can try this step :

Try to turn off the firewall. If the internet works, the solution is to add a new rule in the antivirus program to allow WSA to connect to the internet.

(We will change the firewall mode to let ESET add a new rule for WSA automatically.)

- Open ESET and press F5 to open the advanced setup.
- Go to Network Access Protection.
- Navigate to the Firewall section and in the Filtering Mode, select Learning and click OK.
- Open WSA and access the Google Store or any app that needs an internet connection. The rule should be created automatically.
- Once you finish, go back to ESET in the same section and reset the Filtering Mode to Automatic.
